.travis.yml

```diff
@@ -30,3 +30,4 @@ node_js:
   - "4"
   - "5.6"
   - "5.12"
+  - "6"
```

Comment:
Not sure we want 6 on here, but it gives us visibility to what is failing.

I kept 5.6 on because it fails for some reason and I wanted to make that known.